### DISCUSSION TITLE
Compatibility with this "4"-wire intercom ?

### DISCUSSION THREAD

**Discussion by jpbaril** [PricelessToolkit/ESPBell-LITE#17] [Q&A] [answered]
Hi,
I'm very interested in this board but I don't know anything about electronics and electric circuits.
Is the lite version compatible with this intercom ?
As it seems so close to what is described in the YouTube video I would imagine so, but I prefer to be sure.
https://mircom.com/wp-content/uploads/product_documents/LT-623_IS_51_81_401B_and_489_Wiring_Diagram.pdf

![PXL_20240614_212308418](https://github.com/PricelessToolkit/ESPBell-LITE/assets/5779633/6f1b59af-5623-438a-9c75-9cebe9e70157)
![PXL_20240614_225234712~2](https://github.com/PricelessToolkit/ESPBell-LITE/assets/5779633/421bf45d-efe2-4530-a5b4-f3c34982fe3d)

Thank you

**Comment by PricelessToolkit** [comment]
Hi, to complete the puzzle, I need to know where the doorbell sound is coming from. Do you have a separate chime?

**Reply by jpbaril** [reply]
I'm not sure to understand the question. What do you mean by "where the doorbell sound is coming from" ?
The actual sound gets out of the speaker that we can see in the two previously shown pictures.
The exterior doorbell buttons look like in below picture:
![PXL_20240614_203154748~2](https://github.com/PricelessToolkit/ESPBell-LITE/assets/5779633/a9066eef-d838-4e41-a4c5-9cdacfb38fe2)

**Reply by PricelessToolkit** [reply, marked as answer]
Would you happen to have experience with ESPHome flashing firmware and configuring YAML? if yes there is a 99% chance it will work.

Connection diagram

Intercom | ESPBell-Lite
     1         to      LOCK "2 pair wires"
     3         to      LOCK "2 pair wires"
     2         to      BELL
     4         to      BELL

For powering ESPBell you need an external 5v power supply. (ESPBell power wires marked as GND and VCC)

**Reply by jpbaril** [reply]
I received my ESPBell-LITE.

So, if I understand correctly by looking at [wiring diagram](https://raw.githubusercontent.com/PricelessToolkit/ESPBell-LITE/main/img/doorbell_wiring.jpg) my current wiring will stay as is: I do NOT unplug current wires so to have ESPBell-LITE sits in-between the doorbell and chime box. Instead, ESPBell-LITE will simply sits in parallel to current wires, I only have to connect it in addition to existing. Right ?

![PXL_20240627_225827838~2](https://github.com/PricelessToolkit/ESPBell-LITE/assets/5779633/f21b73bd-783d-43ec-8a4f-a9cdd18706c1)

So, based on wires' color in my picture above, I understand that I have to connect:
- One pair of wires labelled "LOCK" to my intercom/chime box screw # 1. Does it matter which pair ? (i.e Brown+Gray or Red+Black?)
- The other pair of wires labelled "LOCK" to my intercom/chime box screw # 3.
- One of wire labelled "BELL" to my intercom/chime box screw # 2. Same thing, does it matter to which "BELL" wire? (i.e Yellow or Green?)
- The other wire labelled "BELL" to my intercom/chime box screw # 4.

Thanks again for that wonderful product and your help!
(P.S. J'ai vu que le colis provenait de France, donc tu peux me répondre en français si tu préfères)

**Reply by PricelessToolkit** [reply]
Salut :) Mieux vaut en anglais, pour que les autres comprennent, cela pourrait aider quelqu'un.

Yes, that's correct. Your understanding is right. The ESPBell-LITE will simply sit in parallel to the current wires, and you do not need to unplug the existing ones. You just need to connect it in addition to the existing setup. Polarity doesn't matter.

PS: Thank you for your kind words about the product!

**Reply by PricelessToolkit** [reply]
Hi, did you manage to connect and test?

**Reply by jpbaril** [reply]
Not yet. Going to do that on  Monday with my engineer brother as I need to bring (fish) power cables from a light switch that is 30 cm away from the Intercom.
Will let you know of the results.

**Reply by jpbaril** [reply]
Almost 5 months later, I finally tried to connect it. (Had to change my source of power)
The results: it works!

1. The only thing that did not work was that I had to use `password: "password"` instead of `password: "54699445e0aab07e709ffadssd188eb0"` even though my device was from before 2024-10-01.

2. One weird thing, the device's entities were not named "binary_sensor.espbell_lite_doorbell" and "switch.espbell_lite_lock" as per what is seen in the automation example and as I would have expected.
Instead they were simply "binary_sensor.doorbell" and "switch.lock".
Any idea why?

3. And two similar questions:
   1. I understand that this value is the time the door will stay unlocked?
   https://github.com/PricelessToolkit/ESPBell-LITE/blob/c8442ab711ed89999e4effb9c89b5ca83801d21a/Code/ESPHome/espbell-lite.yaml#L80-L81
   2. And these, what are they ? Are they some kind of delay before sending the notification?
   https://github.com/PricelessToolkit/ESPBell-LITE/blob/c8442ab711ed89999e4effb9c89b5ca83801d21a/Code/ESPHome/espbell-lite.yaml#L65-L66



Again, thank you so much for this marvel!
Merci, c'est juste fantastique!

**Reply by PricelessToolkit** [reply]
Hi, 

2. This is normal because, in YAML, names are written like this:  

```yaml
name: "DoorBell"
name: "Lock"
```
You can change it to:

```yaml
name: "${name} DoorBell"  # This will look like binary_sensor.espbell-lite.DoorBell
name: "${name} Lock"      # This will look like switch.espbell-lite.Lock

```


4. Yes, - delay: 1500ms is the duration the door "Relay" stays unlocked. delayed_on: 100ms is a filter for the doorbell input, meaning it will not trigger if the impulse is less than 100ms.

Thank you for supporting my work!

**Reply by elfuerte21** [reply]
Im interested im in usa how i can orden

**Reply by jpbaril** [reply]
<img width="1602" height="1195" alt="Capture d’écran du 2025-11-04 19-34-10" src="https://github.com/user-attachments/assets/49f71252-49f4-4d38-a70c-d462616a39e8" />

**Reply by elfuerte21** [reply]
![Screenshot_20251104_210745_Chrome.jpg](https://github.com/user-attachments/assets/f92681e4-5d94-462d-828e-9e4094c317e6)

**Reply by elfuerte21** [reply]
Its no avaliable to ship to usa

**Reply by PricelessToolkit** [reply]
> Its no avaliable to ship to usa

Yes, it is now impossible to send a parcel from France to USA due to the new tax system.

**Reply by elfuerte21** [reply]
Ok i will buy esp8266 wifi board and you help me to create one like this i will pay you send me contact email to figure how can we do please
